app/src/main/java/org/oppia/app/ongoingtopiclist/OngoingTopicItemViewModel.kt

```diff
@@ -1,7 +1,32 @@
 package org.oppia.app.ongoingtopiclist
 
+import androidx.appcompat.app.AppCompatActivity
 import androidx.lifecycle.ViewModel
+import org.oppia.app.home.RouteToTopicListener
 import org.oppia.app.model.Topic
+import org.oppia.app.shim.IntentFactoryShim
+import org.oppia.app.viewmodel.ObservableViewModel
 
 /** [ViewModel] for displaying topic item in [OngoingTopicListActivity]. */
-class OngoingTopicItemViewModel(val topic: Topic, val entityType: String) : ViewModel()
+class OngoingTopicItemViewModel(
+  val activity: AppCompatActivity,
```

Comment:
Here & elsewhere: please make values private if they don't need to be accessed outside their contained classes.

Reply:
Updated.